JAICore/jaicore-ml/src/test/java/jaicore/ml/scikitwrapper/ScikitLearn_Wrapper_Test.java

```diff
@@ -71,7 +67,6 @@ public void trainClassifierCategorical() throws Exception {
 		ScikitLearnWrapper slw = new ScikitLearnWrapper(constructInstruction, ScikitLearnWrapper.getImportString(imports));
 		Instances dataset = this.loadARFF("testsrc/ml/scikitwrapper/dataset_31_credit-g.arff");
 		slw.buildClassifier(dataset);
-		// assertThat(slw.getModelPath(), not(equalTo("")));
```

Comment:
This test does not check anything but only fails on an exception. What is it that we actually want to test here?

Reply:
Added reasonable asserts.